Command/RemoveThumbsCommand.php

```diff
@@ -236,4 +257,27 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * @return QuestionHelper|DialogHelper
+     */
+    private function getQuestionHelper()
+    {
+        // NEXT_MAJOR: Remove this BC code for SensioGeneratorBundle 2.3/2.4 after dropping support for Symfony 2.3
+        if (class_exists('Sensio\Bundle\GeneratorBundle\Command\Helper\DialogHelper')) {
+            $questionHelper = $this->getHelper('dialog');
+            if (!$questionHelper instanceof DialogHelper) {
```

Comment:
@Soullivaneuh @greg0ire Or maybe it is better to remove `new DialogHelper` and `new QuestionHelper` sections? Are they really neccesary? I took this code fragment from here https://github.com/sonata-project/SonataAdminBundle/pull/4339